app/services/bulk_update/recommendations_uploads/validate_trainee.rb

```diff
@@ -12,6 +12,8 @@ class ValidateTrainee
         provider_trainee_id: "provider trainee ID",
       }.freeze
 
+      OPTIONAL_FORMS = [::PlacementsForm].freeze
```

Comment:
Yes, also since the `Submissions::MissingDataValidator` is been used for qts recommendation from what I've seen and `placements` are not required, not sure why we have [missing_data_validator.rb#L1](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/forms/submissions/missing_data_validator.rb#L16)

Reply:
I think from this morning's meeting we can say that having placements as optional is fine for now. If that changes we will revisit this.